spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyFetcher.java

```diff
@@ -84,7 +84,7 @@ private Object extractBodyFromResponseEntity(ResponseEntity<?> responseEntity) {
         if (responseEntity == null) {
             errorMessage = Optional.of("The ResponseEntity was null.");
         } else if (responseEntity.getBody() == null) {
-            errorMessage = Optional.of("The body was null");
+            errorMessage = Optional.of("The API key could not be found.");
```

Comment:
The previous message was very specific while the new one is very generic. Maybe combine?
```suggestion
            errorMessage = Optional.of("The API key could not be found in the response body, which was 'null'.");
```